extensions/elixir/src/language_servers/next_ls.rs

```diff
@@ -20,8 +20,12 @@ impl NextLs {
     pub fn language_server_binary_path(
         &mut self,
         language_server_id: &LanguageServerId,
-        _worktree: &zed::Worktree,
+        worktree: &zed::Worktree,
     ) -> Result<String> {
+        if let Some(path) = worktree.which("elixir-ls") {
```

Comment:
Similarly here, should we be looking for a NextLS binary instead of `elixir-ls`?

Reply:
Same as above; fixed https://github.com/zed-industries/zed/compare/0cb591cdff769c50ef3c950094a059307155ebb2..c51e3a86fdb76c853b64a07e85e8f7a59e7a1091